spec/redis-lock_spec.rb

```diff
@@ -123,18 +123,40 @@
   end
 
   context "#locked?" do
+    context "when the lock is held" do
+      before :each do
+        lock.lock
+      end
+
+      after :each do
+        lock.unlock
+      end
+
+      it "returns true" do
+        expect(lock.locked_by_me?).to be_truthy
```

Comment:
these tests aren't actually testing the `locked?` method, but if you change `locked_by_me?` to `locked?`, I think we'll be good